libraries/helpers_windows.rb

```diff
@@ -66,9 +71,9 @@ def build_rule(new_resource)
         if new_resource.direction.to_sym == :out
           parameters['localip'] = new_resource.source ? fixup_cidr(new_resource.source) : 'any'
           parameters['localport'] = new_resource.source_port ? port_to_s(new_resource.source_port) : 'any'
-          parameters['interfacetype'] = new_resource.source_interface ? new_resource.source_interface : 'any'
+          parameters['interfacetype'] = new_resource.interface ? new_resource.interface : 'any'
           parameters['remoteip'] = new_resource.destination ? fixup_cidr(new_resource.destination) : 'any'
-          parameters['remoteport'] = port_to_s(new_resource.dest_port) ? new_resource.dest_port : 'any'
+          parameters['remoteport'] = new_resource.dest_port ? port_to_s(new_resource.dest_port) : 'any'
```

Comment:
I'm not sure I understand why you've swapped source_interface with interface, and re-arranged these here. Could you explain? This looks like a breaking change.